gensrc/thrift/Types.thrift

```diff
@@ -687,6 +687,7 @@ enum TSortType {
 enum TMetadataType {
   ICEBERG,
   BACKENDS,
+  FRONTENDS,
```

Comment:
Never add enum definition in the middle of the existing thrift definition. https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=240883637

Reply:
done